watertap/unit_models/gac.py

```diff
@@ -851,7 +867,7 @@ def eq_approx_saturation(b):
                 doc="Molecular diffusion coefficient",
             )
 
-            # TODO: Determine whether the LeBas method can be implemented
+            # TODO: Determine whether the LeBas method can be implemented or embed in prop pack
```

Comment:
Yes, this is something that should be in a property model, but it can be resolved in a future PR. If this isn't in an issue, please add it.

Reply:
I think molal_volume may be similarly added to the property model. I will create the issue now.